tests/limit3/checkresult

```diff
@@ -4,22 +4,32 @@ import sys
 
 max = 50.0
 min = -50.0
+maxvel = 4000
+maxacc = 10000
 
 result_filename = sys.argv[1]
 result_file = open(result_filename, 'r')
 
 retval = 0
 
 for line in result_file.readlines():
-    (line, in_val, out) = line.split()
+    (line, in_val, out, vel, acc) = line.split()
     out = float(out)
+    vel = float(vel)
+    acc = float(acc)
 
-    if float(out) > max:
+    if float(out) > max * 1.01:
         print "max=%.3f, in=%.3f, out=%.3f, violation on line %s" % (max, float(in_val), out, line)
         retval = 1
 
-    if float(out) < min:
+    if float(out) < min * 1.01:
         print "min=%.3f, in=%.3f, out=%.3f, violation on line %s" % (min, float(in_val), out, line)
         retval = 1
 
+    if vel > maxvel:
+        print "maxvel=%.3f, vel=%.3f, violation on line %s" % (maxvel, vel, line)
+
+    if acc > maxacc:
```

Comment:
I realized this should be `if abs(acc) > maxacc:`, and should set `retval = 1`.  (The above `vel` check needs the same.)  There's actually one violation that will cause tests to fail.  I'll post a patch once I figure it out.